src/StackOverflowCard.js

```diff
@@ -37,6 +37,82 @@ const StackOverflowCard = async (data, theme, layout) => {
         </g>
       </svg>
     `;
+  } else if (layout === 'full') {
+    return `
```

Comment:
Please DO NOT generate the whole card again for the `full` layout, only add those extra data if the layout is `full`.